http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/LeoPubsubMessageSubscriber.scala

```diff
@@ -205,6 +217,58 @@ class LeoPubsubMessageSubscriber[F[_]: Async: Timer: ContextShift: Concurrent](
         } yield ()
     }
   }
+
+  private[monitor] def handleDeleteRuntimeMessage(msg: DeleteRuntimeMessage, now: Instant)(
+    implicit traceId: ApplicativeAsk[F, TraceId]
+  ): F[Unit] =
+    for {
+      runtimeOpt <- clusterQuery.getClusterById(msg.runtimeId).transaction
+      runtime <- runtimeOpt.fold(
+        Async[F].raiseError[Runtime](PubsubHandleMessageError.ClusterNotFound(msg.runtimeId, msg))
+      )(Async[F].pure)
+      _ <- if (runtime.status != RuntimeStatus.Deleting)
```

Comment:
just curious, when do we set runtime's state to `Deleting`?

Reply:
So front-leo delete/stop/start runtime sets the DB status to `Deleting`/`Stopping`/`Starting` and publishes a message. The pubsub subscriber (back-Leo) handles the message asynchronously.

Reply:
gotcha..so same way as creating

Reply:
yup